src/util-logopenfile.c

```diff
@@ -416,6 +417,12 @@ int SCConfLogOpenRedis(ConfNode *redis_node, LogFileCtx *log_ctx)
             SCLogError(SC_ERR_MEM_ALLOC, "Unable to allocate redis key command");
             exit(EXIT_FAILURE);
         }
+    } else if (!strcmp(redis_mode, "rlist")) {
+        log_ctx->redis_setup.command = redis_rpush_cmd;
+        if (!log_ctx->redis_setup.command) {
```

Comment:
how can this fail?

Also, why is this in the 3.2 version but not the master version?